examples/nlp_seq/train.py

```diff
@@ -106,6 +108,7 @@ def create_optimizer(model, learning_rate):
       eps=1e-9,
       weight_decay=FLAGS.weight_decay)
   optimizer = optimizer_def.create(model)
+  # TODO(jheek): remove optimizer.replicate
```

Comment:
I think this change came in by accident?

Reply:
oh, yes! removing it...